packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/components/_app-config/index.jsx.hbs

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Should we put a command in current `_app-config/index.jsx` to remind developers to always update this file if we make any change in the code. We want to make sure the this file is always update to date with latest code change in the react-app?